lib/howl/command.moon

```diff
@@ -158,7 +158,8 @@ class CommandInput
     tab: => @command_completion!
     up: => @run_historical!
     ctrl_r: => @run_historical!
-    escape: => self.finish!
+    binding_for:
+      ["cancel"]: => self.finish!
```

Comment:
Out of curiosity, is there a reason this can't just be `cancel: => self.finish!`?